lib/charts/cartesian_renderers/stackedline_chart_renderer.dart

```diff
@@ -0,0 +1,344 @@
+          filter = filterForColumn(column),
+          styles = stylesForColumn(column),
+          fill = new SvgElement.tag('path'),
+          stroke = new SvgElement.tag('path'),
+          fillData = d,
+          // Second alf contains the accumulated data for this measure
+          strokeData = d.sublist(x.length, d.length);
+      e.attributes
+        ..['stroke'] = color
+        ..['fill'] = color
+        ..['class'] = styles.isEmpty
+          ? 'stacked-line-rdr-line'
+          : 'stacked-line-rdr-line ${styles.join(' ')}'
+        ..['data-column'] = '$column';
+      fill.attributes
+        ..['d'] = fillLine.path(fillData, i, e)
+        ..['stroke'] = 'none';
+      stroke.attributes
+        ..['d'] = strokeLine.path(strokeData, i, e)
+        ..['fill'] = 'none';
+      e.children = [fill, stroke];
+      if (isNullOrEmpty(filter)) {
+        e.attributes.remove('filter');
+      } else {
+        e.attributes['filter'] = filter;
+      }
+    });
+
+    if (area.state != null) {
+      svgLines
+        ..on('click', (d, i, e) => _mouseClickHandler(d, i, e))
+        ..on('mouseover', (d, i, e) => _mouseOverHandler(d, i, e))
+        ..on('mouseout', (d, i, e) => _mouseOutHandler(d, i, e));
+    }
+
+    svgLines.exit.remove();
+  }
+
+  @override
+  void dispose() {
+    _disposer.dispose();
+    if (root == null) return;
+    root.selectAll('.stacked-line-rdr-line').remove();
+    root.selectAll('.stacked-line-rdr-point').remove();
+  }
+
+  @override
+  Extent get extent {
+    assert(area != null && series != null);
+    var rows = area.data.rows,
+        max = SMALL_INT_MIN,
+        min = SMALL_INT_MAX,
+        rowIndex = 0;
+
+    rows.forEach((row) {
+      var bar = null;
+      series.measures.forEach((idx) {
+        var value = row.elementAt(idx);
+        if (value != null && value.isFinite) {
+          if (bar == null) bar = 0;
+          bar += value;
+        }
+      });
+      if (bar > max) max = bar;
+      if (bar < min) min = bar;
+      rowIndex++;
+    });
+
+    return new Extent(min, max);
+  }
+
+  @override
+  void handleStateChanges(List<ChangeRecord> changes) {
+    var lines = host.querySelectorAll('.stacked-line-rdr-line');
+    if (lines == null || lines.isEmpty) return;
+
+    for (int i = 0, len = lines.length; i < len; ++i) {
+      var line = lines.elementAt(i),
+          column = int.parse(line.dataset['column']),
+          filter = filterForColumn(column);
+      line.classes.removeAll(ChartState.COLUMN_CLASS_NAMES);
+      line.classes.addAll(stylesForColumn(column));
+      line.attributes['stroke'] = colorForColumn(column);
+      line.attributes['fill'] = colorForColumn(column);
+
+      if (isNullOrEmpty(filter)) {
+        line.attributes.remove('filter');
+      } else {
+        line.attributes['filter'] = filter;
+      }
+    }
+  }
+
+  void _createTrackingCircles() {
+    var linePoints = root.selectAll('.stacked-line-rdr-point')
```

Comment:
Part of the reversing the stack to have lower index measure on top (to match the stacked bar chart)
var linePoints = root.selectAll('.stacked-line-rdr-point')
        .data(series.measures.toList().reversed);